README.md

```diff
@@ -15,17 +15,20 @@ pip install --upgrade postgrid-python
 
 ### Requirements
 
-- Python 3.6+
+-   Python 3.6+
 
 ## Usage
 
-You must have a PostGrid Print & Mail Account (you can sign up at https://dashboard.postgrid.com/signup) and access to your API key which you can retrieve from the [settings page](https://dashboard.postgrid.com/settings).
+To access the Print & Mail services, you must have a PostGrid Print & Mail Account (you can sign up at https://dashboard.postgrid.com/signup) and access to your API key which you can retrieve from the [settings page](https://dashboard.postgrid.com/settings).
+
+Similarly, to access the Address Verification services, you must have a PostGrid Address Verification Account (you can sign up at https://app.postgrid.com/signup) and access to your API key which you can retrieve from the [developers page](https://app.postgrid.com/dashboard/developers).
 
 ```python
 import postgrid
 
 # Swap this out for your live API key to create live orders
 postgrid.pm_key = 'test_sk_...'
+postgrid.av_key = 'test_sk_...'
```

Comment:
Maybe move the AV examples to a different code block so it is more clear that you don't need both keys to use the SDK.